### PR TITLE
ACS Insights tab v2

### PR DESCRIPTION
**Pull request by diegora-ms**
New version of the Azure Communication Services Insights tab

- Followed the Azure Workbooks team's advice and made the text more informational when queries return no results:
image

- Improved the text descriptions in some sections:
image

- Added drill-down functionality to improve dashboard navigation experience:
  - On grids:
image
image
image

  - And on pie charts:
image
image

- Added drill-down for specific IDs, users, or meetings:
image
image

## PR Checklist

* [X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [X] post a screenshot of templates and/or gallery changes
* [X] ensure your template has a corresponding gallery entry in the gallery folder
* [X] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [X] ensure all steps have meaningful names
* [X] ensure all parameters and grid columns have display names set so they can be localized
* [X] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [X] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [X] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [X] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Communication Services/Overview/Overview.workbook [hunk 1]
```diff
@@ -360,50 +364,125 @@
...
-                    "queryType": 0,
-                    "resourceType": "Microsoft.Communication/CommunicationServices"
+                    "version": "NotebookGroup/1.0",
+                    "groupType": "editable",
+                    "title": "Stream health",
+                    "items": [
+                      {
+                        "type": 1,
+                        "content": {
+                          "json": "Streams are classified as healthy if they exhibit telemetry values within the following ranges:\r\n\r\n- Round trip time under 500 miliseconds\r\n- Packet loss rate of less than 10%\r\n- Jitter under 30 miliseconds"
```
Comment (vincentg) [inline Workbooks/Communication Services/Overview/Overview.workbook:416]: Typos   miliseconds => milliseconds (double 'l')

File: Workbooks/Communication Services/Overview/Overview.workbook [hunk 2]
```diff
@@ -884,135 +2406,825 @@
           {
             "type": 1,
             "content": {
-              "json": "The following visualizations represent data from the Chat Operational logs."
+              "json": "The following visualizations represent data from the SMS Operational logs.\r\n\r\n**Click on any part of the pie charts below to visualize filtered logs.**"
             },
-            "name": "chat-text"
+            "name": "auth-text"
```
Comment (vincentg) [inline Workbooks/Communication Services/Overview/Overview.workbook:2411]: should it be sms-text ?

**Review by vincentg — CHANGES_REQUESTED** [review]
Minor comment for Typos

File: Workbooks/Communication Services/Overview/Overview.workbook (hunk 2 above)
Comment (diegora-ms) [reply Workbooks/Communication Services/Overview/Overview.workbook]: Great catch! Fixed!

File: Workbooks/Communication Services/Overview/Overview.workbook (hunk 1 above)
Comment (diegora-ms) [reply Workbooks/Communication Services/Overview/Overview.workbook]: You're right, fixed this too.

**Review by vincentg — APPROVED** [review]
Thanks!  LGTM

**Pull request merged**